defender/staged/downloaded/Forta_cToken.js

```diff
@@ -0,0 +1,207 @@
+  const { request } = autotaskEvent;
+  if (request === undefined) {
+    throw new Error('request undefined');
+  }
+
+  // ensure that the body key exists within the request Object
+  const { body } = request;
+  if (body === undefined) {
+    throw new Error('body undefined');
+  }
+
+  // ensure that the alert key exists within the body Object
+  const { alert } = body;
+  if (alert === undefined) {
+    throw new Error('alert undefined');
+  }
+
+  // ensure that the alert key exists within the body Object
+  const { source } = body;
+  if (source === undefined) {
+    throw new Error('source undefined');
+  }
+
+  // extract the metadata from the alert Object
+  const { metadata } = alert;
+  if (source === undefined) {
+    throw new Error('metadata undefined');
+  }
+
+  // extract the hashes from the source Object
+  const {
+    transactionHash,
+  } = source;
+
+  // Start of usual modifications to the autotask script
+  // extract the metadata
+  const {
+    cTokenSymbol,
+    eventName,
+    usdValue,
+  } = metadata;
+  if (usdValue === undefined) {
+    throw new Error('usdValue undefined, please use newer bot version');
+  }
+
+  const eventMapping = {
+    Borrow: {
+      amountKey: 'borrowAmount',
+      byKey: 'borrower',
+      action: 'borrow',
+    },
+    LiquidateBorrow: {
+      amountKey: 'repayAmount',
+      byKey: 'liquidator',
+      fromKey: 'borrower',
+      action: 'liquidate',
+    },
+    Mint: {
+      amountKey: 'mintAmount',
+      byKey: 'minter',
+      action: 'supply',
+    },
+    Redeem: {
+      amountKey: 'redeemAmount',
+      byKey: 'redeemer',
+      action: 'withdraw',
+    },
+    RepayBorrow: {
+      amountKey: 'repayAmount',
+      byKey: 'payer',
+      forKey: 'borrower',
+      action: 'repay',
+    },
+  };
+
+  function emojiForEvent(eventType, usdValueString) {
+    // create the appropriate number of whale emoji for the value
+    // add one whale for each power of 1000
+    const numWhales = Math.floor((usdValueString.length - 1) / 3);
+    const whaleString = '🐳'.repeat(numWhales);
+
+    switch (eventType) {
+      case 'Borrow':
+        return whaleString.concat('📥');
+      case 'LiquidateBorrow':
+        return whaleString.concat('💔');
+      case 'Mint':
+        return whaleString.concat('📈');
+      case 'Redeem':
+        return whaleString.concat('📉');
+      case 'RepayBorrow':
+        return whaleString.concat('📤');
+      default:
+        return '';
+    }
+  }
+
+  const eventEmoji = emojiForEvent(eventName, usdValue);
+
+  const internationalNumberFormat = new Intl.NumberFormat('en-US');
+  const amountString = internationalNumberFormat.format(usdValue);
+
+  const eventObject = eventMapping[eventName];
+  const byAddress = metadata[eventObject.byKey];
+
+  const { action } = eventObject;
+  let message = `${eventEmoji} **$${amountString} of ${cTokenSymbol}** ${action}`;
+
+  if (action === 'liquidate') {
```

Comment:
```suggestion
  if (action === 'liquidate') {
    const lastIndex = message.lastIndexOf(' '); // gets the last index of the last space (before action)
    message = message.substring(0, lastIndex); // changes 'liquidate' to 'liquidated'
    message += ' liquidated'

    const fromAddress = metadata[eventObject.fromKey];
    message += ` from ${fromAddress.slice(0, 6)} by ${byAddress.slice(0, 6)}`;
  } else if (action === 'repayBorrow') {
    const forAddress = metadata[eventObject.forKey];
    message += ` by ${byAddress.slice(0, 6)}`;
    if (forAddress !== byAddress) {
      message += ` for ${forAddress.slice(0, 6)}`;
    }
  } else {
    message += ` by ${byAddress.slice(0, 6)}`;
  }
```

Reply:
added in lines `186 - 188` to change the message from `liquidate` to `liquidated`

Reply:
There was a simpler approach to change the eventMapping on line 143.
addressed in commit dd317691d5cf341884b6731f88bbc50fa6a09e17